torch/distributed/_tensor/device_mesh.py

```diff
@@ -34,12 +34,38 @@
 class _MeshEnv:
     def __init__(self) -> None:
         self.mesh_stack: List[DeviceMesh] = []
+        self.child_to_parent_mapping = {}
 
     def get_current_mesh(self) -> "DeviceMesh":
         if len(self.mesh_stack) == 0:
             raise RuntimeError("No device mesh is currently active!")
         return self.mesh_stack[-1]
 
+    def create_child_mesh(
```

Comment:
are you planning on unifying this util (remove this util) https://github.com/pytorch/pytorch/blob/main/torch/distributed/tensor/parallel/_utils.py#L118

Reply:
> are you planning on unifying this util (remove this util) https://github.com/pytorch/pytorch/blob/main/torch/distributed/tensor/parallel/_utils.py#L118

Yes! Discussed with @fduwjj today. I will do that in a later PR to update TP API as well unifying the util.